src/services/Functional/Video.js

```diff
@@ -8,4 +8,22 @@ export default {
     }
     return { valid: false };
   },
+
+  /**
+   * extracts the video Id from the YouTube URL
```

Comment:
I think the name of the method and the comment on the `videoURL` parameter is enough, no need of this docstring